src/machines/modelingMachine.ts

```diff
@@ -1687,7 +1694,7 @@ export const modelingMachine = setup({
 
         Revolve: {
           target: 'idle',
-          guard: 'has valid sweep selection',
+          guard: 'has valid revolve selection',
           actions: ['AST revolve'],
```

Comment:
In a future PR I'd love to refactor this to be an actor instead of an action. We have examples now from the offset planes point-and-click PR and the loft and shell ones to follow.